dataplane/standalone/common.cc

```diff
@@ -156,5 +159,26 @@ sai_object_id_t AttributeManager::entry_to_switch_id(sai_object_type_t type,
     default:
       throw "Invalid type";
   }
-  return swID;
+  return std::to_string(swID);
+}
+
+sai_status_t APIBase::create(common_entry_t id, uint32_t attr_count,
```

Comment:
input id is not being used

Reply:
the id will be used classes that inherit from this class